### DISCUSSION TITLE
Can pypdf convert a PDF page to image?

### DISCUSSION THREAD

**Discussion by dclong** [py-pdf/pypdf#2164] [Q&A]
I don't find such an example in the doc.

**Comment by pubpub-zz** [comment]
- - Pypdf is. A library to work on pdf éléments. What you are looking for is. A virtual viewer that pypdf is not. Sorry

**Comment by stefan6419846** [comment]
You probably want something like `pdftocairo` or if you are searching for a Python wrapper `pdf2image`.

**Comment by pubpub-zz** [comment]
for follow I convert this thread into a discussion
